src/Imath/ImathExport.h

```diff
@@ -3,15 +3,58 @@
 // Copyright Contributors to the OpenEXR Project.
 //
 
+// See the document in OpenEXR docs/SymbolVisibility.md for background
+// on all these types
 #if defined(IMATH_DLL)
 #    if defined(IMATH_EXPORTS)
 #        define IMATH_EXPORT __declspec(dllexport)
```

Comment:
Would it be useful to preface each of these #define's with #ifndef, to make it possible to overrided with a command-line setting if necessary? In case anyone wants/needs to alter the behavior?

Reply:
well, you can't do that for windows dlls, ever, unless you are doing static libraries (which is the IMATH_DLL thing), but I've added an option to override the behavior